br/pkg/utils/cdc.go

```diff
@@ -0,0 +1,112 @@
+package utils
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"go.uber.org/zap"
+)
+
+const (
+	CDCPrefix      = "/tidb/cdc/"
+	ChangefeedPath = "/changefeed/info/"
+	CDCPrefixV61   = "/tidb/cdc/changefeed/info/"
+)
+
+type CDCNameSet struct {
```

Comment:
lint should complain that this structure has no comments

Reply:
done